app/client/cli/keys.go

```diff
@@ -0,0 +1,478 @@
+	cmd.AddCommand(getCmds...)
+	cmd.AddCommand(exportCmds...)
+	cmd.AddCommand(importCmds...)
+	cmd.AddCommand(signCmds...)
+
+	return cmd
+}
+
+func keysCreateCommands() []*cobra.Command {
+	cmds := []*cobra.Command{
+		{
+			Use:     "Create",
+			Short:   "Create new key",
+			Long:    "Creates a new key and stores it in the keybase",
+			Aliases: []string{"create"},
+			Args:    cobra.ExactArgs(0),
+			RunE: func(cmd *cobra.Command, args []string) error {
+				// Open the debug keybase at the specified path
+				pocketDir := strings.TrimSuffix(dataDir, "/")
+				keybasePath, err := filepath.Abs(pocketDir + keybaseSuffix)
+				if err != nil {
+					return err
+				}
+				kb, err := keybase.NewKeybase(keybasePath)
+				if err != nil {
+					return err
+				}
+
+				if !nonInteractive {
+					pwd = readPassphrase(pwd)
+				}
+
+				kp, err := kb.Create(pwd, hint)
+				if err != nil {
+					return err
+				}
+
+				logger.Global.Info().Str("address", kp.GetAddressString()).Msg("New Key Created")
+
+				return kb.Stop()
+			},
+		},
+	}
+	return cmds
+}
+
+func keysUpdateCommands() []*cobra.Command {
+	cmds := []*cobra.Command{
+		{
+			Use:     "Update <addrHex> [--pwd] [--new_pwd] [--hint]",
+			Short:   "Updates the key to have a new passphrase and hint",
+			Long:    "Updates the passphrase and hint of <addrHex> in the keybase, using either the values from the flags provided or from the CLI prompts.",
+			Aliases: []string{"update"},
+			Args:    cobra.ExactArgs(1),
+			RunE: func(cmd *cobra.Command, args []string) error {
+				// Unpack CLI args
+				addrHex := args[0]
+
+				// Open the debug keybase at the specified path
+				pocketDir := strings.TrimSuffix(dataDir, "/")
+				keybasePath, err := filepath.Abs(pocketDir + keybaseSuffix)
+				if err != nil {
+					return err
+				}
+				kb, err := keybase.NewKeybase(keybasePath)
+				if err != nil {
+					return err
+				}
+
+				if !nonInteractive {
+					pwd = readPassphrase(pwd)
+					newPwd = readPassphraseMessage(newPwd, "New passphrase: ")
+				}
+
+				err = kb.UpdatePassphrase(addrHex, pwd, newPwd, hint)
+				if err != nil {
+					return err
+				}
+
+				logger.Global.Info().Str("address", addrHex).Msg("Key updated")
+
+				return kb.Stop()
+			},
+		},
+	}
+	return cmds
+}
+
+func keysDeleteCommands() []*cobra.Command {
+	cmds := []*cobra.Command{
+		{
+			Use:     "Delete <addrHex>",
+			Short:   "Deletes the key from the keybase",
+			Long:    "Deletes <addrHex> from the keybase",
+			Aliases: []string{"delete"},
+			Args:    cobra.ExactArgs(1),
+			RunE: func(cmd *cobra.Command, args []string) error {
+				// Unpack CLI args
+				addrHex := args[0]
+
+				// Open the debug keybase at the specified path
+				pocketDir := strings.TrimSuffix(dataDir, "/")
+				keybasePath, err := filepath.Abs(pocketDir + keybaseSuffix)
+				if err != nil {
+					return err
+				}
+				kb, err := keybase.NewKeybase(keybasePath)
+				if err != nil {
+					return err
+				}
```

Comment:
@Olshansk So the reason I did this is because of the linter - I can't defer without error checking, I will go through and manually close when the db is no longer "needed" and return nil instead of the db.Stop value